asciidoc/modules/con_mg_changes-in-vertx-mail-client.adoc

```diff
@@ -0,0 +1,12 @@
+[id="changes-in-vertx-mail-client_{context}"]
+= Changes in {VertX} mail client
+
+The following section describes the changes in {VertX} mail client.
+
+== `MailAttachment` is available as an interface
+
+From {VertX} {4} onwards, `MailAttachment` is available as an interface. It enables you to use the mail attachment functionality in a stream. In earlier releases of {VertX}, `MailAttachment` was available as a class and attachment for mails was represented as a data object.
+
+== Mail configuration interface extends the net client options
+
+`MailConfig` interface extends the `NetClientOptions` interface. Due to this extension, mail configuration also supports the proxy setting of the `NetClient`XXX.
```

Comment:
@vietj In the [wiki](https://github.com/vert-x3/wiki/wiki/4.0.0-Deprecations-and-breaking-changes#%EF%B8%8F-mailconfig-extends-netclientoptions) page, we have the following points for this point. Can you please help me understand/rephrase these points?

This will lead to the following breaks:

This will lead to different output Json format regarding to the trusted keystore part, it can read the old Json format configuration.
The hashCode() and equals() methods take consideration of fields in the super class.

Reply:
I think those are technical concerns that won't affect users directly and we can omit this entirely.